### DISCUSSION TITLE
Test logs limit

### DISCUSSION THREAD

**Discussion by talmoshe24** [reportportal/reportportal#1380] [General]
Hi,

I have RP 5.3.3 with JUnit4 integration
I noticed that in a specific test with thousands of log lines only 2560 lines exist in the launch, where Jenkins console shows many more.
Is there a limit on the number of logs per test?

Thanks,
Tal

**Comment by HardNorth** [comment]
Everything regarding this was discussed here: https://github.com/reportportal/client-java/issues/21
